README.rdoc

```diff
@@ -22,9 +22,14 @@ Configure your environment. For example, create an initializer in Rails in <tt>c
     TaxCloud.configure do |config|
       config.api_login_id = 'your_tax_cloud_api_login_id'
       config.api_key = 'your_tax_cloud_api_key'
-      config.usps_username = 'your_usps_username'
+      config.usps_username = 'your_usps_username' # optional
+      config.open_timeout = 1 # optional
+      config.read_timeout = 1 # optional
     end
 
+The <tt>open_timeout</tt> and <tt>read_timeout</tt> options are used to specify a waiting time for the TaxCloud web service
+response. The default values are too high, and if the service is not responding you will have to wait for the response a very long time.
```

Comment:
@drewtempelmeyer , actually the problem is following: TaxCloud closes network connection in case if we don't follow their rules. They even don't inform somehow, don't respond with an exception - just closing connection and that's it. And currently in this case the timeout exception from Savon will be reached within ~30 seconds or more. This is long period of time and I want to be able decrease the waiting time. The added options allow to do this for me.

If you want decrease them by default - ok, let's do this. But what the values should be?